src/twist_controller.cpp

```diff
@@ -108,11 +104,19 @@ MoveitTwistController::on_configure( const rclcpp_lifecycle::State & /*previous_
     RCLCPP_INFO( get_node()->get_logger(), "previous_goal_state_ size: %lu",
                  previous_goal_state_.size() );
 
-    gripper_joint_name_ = params.gripper_joint_name;
-    RCLCPP_INFO( get_node()->get_logger(), "Gripper joint: %s", gripper_joint_name_.c_str() );
+    RCLCPP_INFO( get_node()->get_logger(), "Gripper joint: %s", params_.gripper_joint_name.c_str() );
     if ( !loadGripperJointLimits() ) {
       return controller_interface::CallbackReturn::ERROR;
     }
+
+    // Current limit setup
+    if ( params_.request_current_interface ) {
+      if ( params_.current_limits.size() != arm_joint_names_.size() ) {
```

Comment:
It would be better to set the current limits with a map, e.g.
```
limits:
  arm_joint_1: 1.0
  arm_joint_2: 2.0
```
Otherwise, a mixup in the order could lead to unpredictable behavior.